sky/provision/kubernetes/network.py

```diff
@@ -0,0 +1,221 @@
+import os
+from typing import Any, Dict, List, Optional, Tuple
+
+from sky.provision import common
+from sky.provision.kubernetes import network_utils
+from sky.utils import kubernetes_enums
+from sky.utils import ux_utils
+from sky.utils.resources_utils import port_ranges_to_set
+
+_PATH_PREFIX = "/skypilot/{cluster_name_on_cloud}/{port}"
+_LOADBALANCER_SERVICE_NAME = "{cluster_name_on_cloud}-skypilot-loadbalancer"
```

Comment:
nit - single quotes for strs
```suggestion
_PATH_PREFIX = '/skypilot/{cluster_name_on_cloud}/{port}'
_LOADBALANCER_SERVICE_NAME = '{cluster_name_on_cloud}-skypilot-loadbalancer'
```

Reply:
Same for the rest of the PR